addons/event_sale/views/sale_order_views.xml

```diff
@@ -5,12 +5,12 @@
         <field name="model">sale.order</field>
         <field name="inherit_id" ref="sale.view_order_form" />
         <field name="arch" type="xml">
-            <button name="action_preview_sale_order" position="before">
+            <xpath expr="//sheet/div" position="inside">
```

Comment:
Would it be possible to add hasclass "oe_button_box" ?

"div" is not specific enough and could break in the future.

Reply:
Hello @awa-odoo 

Changes have been done.
Can you review it again.

Thanks :)